flake.lock

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
I don't know how nix works, checking that these files are deliberately added?

Reply:
Yea. At some point we debated having nix dev environment definitions along the project. Meanwhile it's been **very** helpful with handful of other projects we develop (eg. conntest, hashistack, gollector, recovery). These two files **guarantee** that whenever we open up the environment (for example in a years time or later) we get the same working environment that we had the last time we updated the lock file.
Moreover this also adds a few helpful pre-commit hooks. I figured it'd be useful for those who decide to use nix to have it ready in the repo. It's two files that _ladi-dadi-we-like-to-nixify-we-**don't-bother-anybody**_